cmd/podman/utils/utils.go

```diff
@@ -123,17 +123,14 @@ func IsCheckpointImage(ctx context.Context, namesOrIDs []string) (bool, error) {
 	}
 	imgID := imgData[0].ID
 
-	hostInfo, err := registry.ContainerEngine().Info(ctx)
-	if err != nil {
-		return false, err
-	}
+	runtimeName := registry.PodmanConfig().RuntimePath
```

Comment:
Do we need to call filepath.Base() to get name and not the full path?

Reply:
fixed